addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -252,6 +248,12 @@ private void requestPlayerUpdate(int activePlayer, boolean updateMediaType) {
         }
 
         String mediaType = item.get("type").getAsString();
+        if (mediaType.equals("channel") && item.has("channeltype")) {
```

Comment:
Position literals first in String comparisons - that way if the String is null you won't get a NullPointerException, it'll just return false. [source: PMD](http://pmd.sourceforge.net/pmd-4.3.0/rules/design.html#PositionLiteralsFirstInComparisons)